Adyen/Constants/AdditionalData.cs

```diff
@@ -46,5 +46,11 @@ public class AdditionalData
         public const string RecurringShopperReference = "recurring.shopperReference";
         public const string RecurringDetailReference = "recurring.recurringDetailReference";
         public const string AllowThreeDS2 = "allow3DS2";
+        public const string PaymentLinkId = "paymentLinkId";
+        public const string MerchantOrderReference = "merchantOrderReference";
+        // Custom constants for metadata
+        public const string OrderReference = "metadata.orderReference";
+        public const string OrderId = "metadata.orderId";
+        public const string OrderNumber = "metadata.orderNumber";

```

Comment:
@AlexandrosMor `OrderReference`, `OrderId` and `OrderNumber` should probably not be included. These were just custom meta data properties I sent in the request from the ecommerce system, but not something specific in Adyen.

Reply:
Thank you for letting us know :)

Reply:
I think `OrderNumber` should be removed as well as Adyen doesn't return this meta data and this was a custom meta data I sent in my use-case.

As I recall the `reference` property in the request is returned as `merchantOrderReference` in webhook notification.